### DISCUSSION TITLE
Index own catalog eg. with portable programs

### DISCUSSION THREAD

**Discussion by arlido** [Flow-Launcher/Flow.Launcher#1115] [Q&A] [answered]
Hi,

Is it any solution to add to index my own catalog with eg. portable programs? These feature example works well in Launchy. I see that only installed programs are on index, what about enlarge that functionality to add own catalog to search index (example is search for .exe files in windows).

Cheers

**Comment by taooceros** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/45326534/161792078-f9683dc8-92ec-434d-a7bb-45587e1e9a96.png)
